package.json

```diff
@@ -4,7 +4,7 @@
     "android": "expo start --android",
     "eject": "expo eject",
     "ios": "expo start --ios",
-    "lint": "./node_modules/.bin/eslint __tests__ src",
+    "lint": "./node_modules/.bin/eslint __tests__ src --ext .js,.jsx,.ts,.tsx",
```

Comment:
we do not have `jsx` and `tsx` files in the codebase. these two can be removed, right?

Reply:
tbh I did not realise we have not `.jsx` files. 🤣 
I can remove that one, if you want.

We will have `.tsx` files in the project. Everytime we have a functional component it needs to be in a `.tsx` file and not a plain `.ts` file.

Reply:
ok, then keep `.jsx` as well for now. i will read about it later.